app/views/users/new.html.erb

```diff
@@ -1,5 +1 @@
-<h1>New user</h1>
-
-<%= render "form_create", user: @user %><br>
-
-<%= link_to 'Already have an account? Log in.', "/login" %>
+<%= render "./shared/form_create", obj: @user, title: "Create your account", submit_text:"Register", submit_path: users_path, go_to_url:login_path, go_to_message:"Already have an account? Log in." %>
```

Comment:
@JuanMaGiraldo missing blank space closing the file.

Reply:
Fixed, thanks!